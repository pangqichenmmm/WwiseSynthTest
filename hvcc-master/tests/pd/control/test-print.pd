#N canvas 231 101 279 228 10;
#X obj 33 124 t;
#X obj 61 124 t b b;
#X obj 33 32 loadbang;
#X obj 182 179 print B;
#X obj 33 178 print A;
#X obj 106 125 t a a;
#X msg 155 93 0.5;
#X obj 155 124 t f f;
#X msg 76 94 symbol dog;
#X obj 33 54 t b b b b;
#X connect 0 0 4 0;
#X connect 0 1 3 0;
#X connect 1 0 4 0;
#X connect 1 1 3 0;
#X connect 2 0 9 0;
#X connect 5 0 4 0;
#X connect 5 1 3 0;
#X connect 6 0 7 0;
#X connect 7 0 4 0;
#X connect 7 1 3 0;
#X connect 8 0 5 0;
#X connect 9 0 0 0;
#X connect 9 1 1 0;
#X connect 9 2 8 0;
#X connect 9 3 6 0;
