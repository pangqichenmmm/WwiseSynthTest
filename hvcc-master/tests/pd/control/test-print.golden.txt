[@ 0.000] B: 0.5
[@ 0.000] A: 0.5
[@ 0.000] B: symbol dog
[@ 0.000] A: symbol dog
[@ 0.000] B: bang
[@ 0.000] A: bang
[@ 0.000] B: bang
[@ 0.000] A: bang
